public/css/table.css

```diff
@@ -0,0 +1,13 @@
+table {
```

Comment:
Reminder to check that the following props are unaffected by this global CSS:
- padding
- size
- stickyHeader